content/source/docs/enterprise/before-installing/index.html.md

```diff
@@ -73,13 +90,18 @@ Make sure your data storage services or device meet Terraform Enterprise's requi
 - **Mounted disk:**
     - [Mounted Disk Requirements](./disk-requirements.html)
 
+-> **Note:** If you are following one of the [reference architectures](./reference-architecture/index.html), refer to it while preparing your data storage services.
+
 ## Linux Instance
 
-Install Terraform Enterprise on a Linux instance of your choosing.
-You will start and manage this instance like any other server.
+Terraform Enterprise runs on Linux instances. The source of these instances depends on your deployment method:
+
+- **Clustered deployment:** Terraform automatically provisions all instances for Terraform Enterprise. The machine image and the instance type are configurable:
+    - By default, the module uses an official Ubuntu image; you can override this with any image that meets the [software requirements below](#software-requirements).
+    - The default instance type depends on the cloud you deploy to; see the module documentation for details. You can override this, and can optionally specify separate image types for primary and secondary instances.
+- **Individual deployment:** You must prepare a running Linux instance for Terraform Enterprise before running the installer. You will start and manage this instance like any other server.
 
-The Terraform Enterprise installer currently supports the following
-operating systems:
+In either case, note that Terraform Enterprise currently supports running under the following operating systems:
```

Comment:
Clustering only supports Ubuntu, and RHEL 7.2-7.6 at the moment. (RHEL 7.7 is specifically unsupported in the current version of replicated.)

Reply:
👍🏼 !